### PR TITLE
Dockerize Omnia with new SSB RPC

### PR DESCRIPTION
**Pull request by teghnet**

### CODE REVIEW

File: lib/util.sh [hunk 1]
```diff
@@ -85,6 +85,7 @@ signMessage () {
 	for arg in "$@"; do
 		_data+="$arg"
 	done
-	verbose "Signing message..."
-	ethsign message --from "$ETH_FROM" --key-store "$ETH_KEYSTORE" --passphrase-file "$ETH_PASSWORD" --data "$_data"
+	verbose "Signing message..." "from=$ETH_FROM"
+	ethsign message --from "$ETH_FROM" --key-store "$ETH_KEYSTORE" --passphrase-file "$ETH_PASSWORD" \
+		--data "$_data" 2> >(verbose --raw "ethsign log" "$(cat)")
```
Comment (teghnet) [inline lib/util.sh:90]: because ethsign spits out logs

File: lib/source.sh [hunk 2]
```diff
@@ -47,21 +47,28 @@ readSourcesWithSetzer()  {
...
-				[ ..
-				| select(type == "object" and .type == "origin" and .error == null)
-				| {(.base+"/"+.quote+"@"+.params.origin): (.price|tostring)}
-				]
-				| add
-			)
-		}
-	' <<<"$(gofer price --config "$GOFER_CONFIG" --format json "$@")")"
+	local _data;
+	if _data=$(gofer price --config "$GOFER_CONFIG" --format json "$@")
```
Comment (teghnet) [inline lib/source.sh:51]: We were masking gofer error messages

File: bin/omnia [hunk 3]
```diff
@@ -11,7 +11,6 @@ source ./config.sh
 source ./gasprice.sh
 source ./ethereum.sh
 source ./log.sh
-source ./scuttlebot.sh
```
Comment (teghnet) [inline bin/omnia:14]: obsolete

File: docker-compose.yml [hunk 4]
```diff
@@ -12,106 +12,142 @@ services:
...
+     - "8002:8000"
    volumes:
-     - "./docker/ssb-server/config/config_server_relay.json:/root/.ssb/config"
-     - "./docker/ssb-server/config/secret_relay:/root/.ssb/secret"
+     - "./docker/ssb-server/config/relay_config.json:/root/.ssb/config"
+     - "./docker/ssb-server/config/relay_secret:/root/.ssb/secret"
+
+  gofer:
+    container_name: gofer.local
+    image: ghcr.io/chronicleprotocol/oracle-suite:dev
```
Comment (teghnet) [inline docker-compose.yml:73]: single `Go` development image

File: docker/gofer/client.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{"ethereum": {
+  "rpc": "http://geth.local:8545"
```
Comment (teghnet) [inline docker/gofer/client.json:2]: I am not fond of the idea that a thin client needs that. 
I propose to create a new binary with only the client - e.g. `gofer` & `goferd`

File: lib/config.sh [hunk 6]
```diff
@@ -1,5 +1,5 @@
 importEnv () {
-	local _here=$(cd "${BASH_SOURCE[0]%/*}" && pwd)
+	local _here;_here=$(cd "${BASH_SOURCE[0]%/*}" && pwd)
```
Comment (teghnet) [inline lib/config.sh:2]: errors were masked

**Review by konstantinzolotarev — APPROVED** [review]

**Pull request merged**